sourced/ml/mining/models/stdlib.py

```diff
@@ -0,0 +1,63 @@
+from modelforge import merge_strings, Model, register_model, split_strings
+from sourced.ml.core.models.license import DEFAULT_LICENSE
+
+
+@register_model
+class StandardLibraries(Model):
+    """
+    List of standard library contents for each Babelfish language.
+    """
+
+    NAME = "stdlib"
+    VENDOR = "source{d}"
+    DESCRIPTION = (
+        "The list of standard library contents for C++, C#, Go, Java, JavaSript, PHP, "
+        "Python and Ruby. Language version is not taken into account, data was collected "
+        "as exhaustively as possible."
+    )
+    LICENSE = DEFAULT_LICENSE
+
+    def construct(self, libraries, infos):
+        self._libraries = libraries
+        self._infos = infos
```

Comment:
I still wonder what "infos" means... We should have a better name.

Reply:
yeah, I agree it's not super clear. as it's categorical information of various nature, I think `metadata` or `tags` is appropriate. I will replace with the former, but if you'd prefer something else tell me after